simple/stats/runner.py

```diff
@@ -57,62 +48,69 @@ class Runner:
     """
 
   def __init__(self,
-               config_file: str,
-               input_dir: str,
-               output_dir: str,
+               config_file_path: str,
+               input_dir_path: str,
+               output_dir_path: str,
                mode: RunMode = RunMode.CUSTOM_DC) -> None:
-    assert config_file or input_dir, "One of config_file or input_dir must be specified"
-    assert output_dir, "output_dir must be specified"
+    assert config_file_path or input_dir_path, "One of config_file or input_dir must be specified"
+    assert output_dir_path, "output_dir must be specified"
 
     self.mode = mode
-    self.input_handlers: list[FileHandler] = []
+
+    # New option to traverse subdirs of input dir(s).
+    self.include_input_subdirs = bool(os.getenv("INCLUDE_INPUT_SUBDIRS"))
```

Comment:
Thoughts on basing it on a property in config.json vs in an env variable?

Reply:
I chose an env variable because it's similar to the approach we use for database config (e.g. `ENV_USE_CLOUDSQL`). Is there a principle you'd recommend for deciding what goes in config.json vs an env variable?

Reply:
A good demarcation is values that are used by the services container vs only by the importer. This falls in the latter category so leaning towards config.json.